tools/code_format/check_format.py

```diff
@@ -567,7 +567,7 @@ def tokenInLine(token, line):
   index = 0
   while True:
     index = line.find(token, index)
-    if index < 1:
+    if index < 0:
```

Comment:
Mind to put your comment in the commit description here too? Plus if you can add a case example here will be good. Thanks!

Reply:
@dio Just to clarify, do you mean putting the example case in the comment section?

Reply:
Yes, sorry for not very clear earlier. Put an inline comment here, why we need to branch when the index is 1 (and not 0), and what is the implication as you have described in the PR description above.

Reply:
Comments are added. Let me know if this is okay now.